### ISSUE TITLE
Prevent editor elements from editing

### ISSUE THREAD

**Issue by seszcze** [yabwe/medium-editor#1442]
Is it possible to prevent editor elements from editing?

Let's say this is my content with medium editor
<img width="1165" alt="1eda16a9f0f25caeca97b82e43f88835" src="https://user-images.githubusercontent.com/6885572/39077675-0055e71c-4504-11e8-899b-35a08d5583b3.png">

and I would like to block table.

**Comment by herzinger** [comment]
Yes. Just add `contenteditable="false"` (the html attribute) to the table element.
